routes/twill.php

```diff
@@ -1,123 +1,123 @@
-    Route::module('artworks');
+    TwillRoutes::module('artworks');
     Route::get('artworks/augment/{datahub_id}', [ArtworkController::class, 'augment'])->name('collection.artworks.augment');
-    Route::module('artists');
+    TwillRoutes::module('artists');
     Route::get('artists/augment/{datahub_id}', [ArtistController::class, 'augment'])->name('collection.artists.augment');
 
     Route::group(['prefix' => 'interactive_features'], function () {
-        Route::module('interactiveFeatures');
-        Route::module('experiences');
-        Route::module('experiences.slides');
+        TwillRoutes::module('interactiveFeatures');
+        TwillRoutes::module('experiences');
+        TwillRoutes::module('experiences.slides');
     });
 
-    Route::module('authors');
+    TwillRoutes::module('authors');
 
-    Route::module('categoryTerms');
+    TwillRoutes::module('categoryTerms');
     Route::get('categoryTerms/augment/{datahub_id}', [CategoryTermController::class, 'augment'])->name('collection.categoryTerms.augment');
 
     Route::group(['prefix' => 'research_resources'], function () {
         Route::get('landing', [PageController::class, 'research'])->name('collection.research_resources.landing');
-        Route::module('educatorResources');
+        TwillRoutes::module('educatorResources');
     });
 
     Route::group(['prefix' => 'articles_publications'], function () {
         Route::get('landing', [PageController::class, 'articles_publications'])->name('collection.articles_publications.landing');
         Route::get('articles_landing', [PageController::class, 'articles'])->name('collection.articles_publications.articles_landing');
-        Route::module('articles');
-        Route::module('categories');
-        Route::module('videos');
-        Route::module('printedPublications');
-        Route::module('digitalPublications');
-        Route::module('digitalPublications.articles');
+        TwillRoutes::module('articles');
+        TwillRoutes::module('categories');
+        TwillRoutes::module('videos');
+        TwillRoutes::module('printedPublications');
+        TwillRoutes::module('digitalPublications');
+        TwillRoutes::module('digitalPublications.articles');
 
         // WEB-1963: Browser for nested modules must be implemented manually
         Route::get('/digitalPublicationsBrowser/articles/browser?digitalPublication={digitalPublication}', [DigitalPublicationArticleController::class, 'browser'])->name('collection.articles_publications.digitalPublications.articles.subbrowser');
-        Route::get('/digitalPublicationsBrowser/articles/browser', [DigitalPublicationArticleController::class, 'browser'])->name('collection.articles_publications.digitalPublications.articles.browserbrowser');
+        Route::get('/digitalPublicationsBrowser/articles/browser', [DigitalPublicationArticleController::class, 'browser'])->name('collection.articles_publications.digitalPublications.articles.browser');
     });
```

Comment:
@nikhiltri this is the only change I didn't recognize from your merge conflict resolutions. Either the [reference to this route name](https://github.com/art-institute-of-chicago/artic.edu/blob/feature/twill-index-table-columns/resources/views/twill/partials/featured-related.blade.php#L35) needs to be update, or this needs to be reverted.